code/style.css

```diff
@@ -1,7 +1,205 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
```

Comment:
This file needs some work on the indention! 😀

Reply:
Thank you! What is the CSS indentation supposed to look like?

Reply:
Same way as the html and js. It should be nested of each other
`.my-class {` is where we declare what element we want to change
`  background-color: red;` inside we do the changes for that element
`}` then we close the element

So it becomes 
```
.my-class {
  background-color: red;
}
```